src/timeMachine/components/SubmitQueryButton.test.tsx

```diff
@@ -159,7 +159,7 @@ describe('TimeMachine.Components.SubmitQueryButton', () => {
 
     fireEvent.click(getByTitle('Submit'))
     expect(getByTitle('Submit')).toBeTruthy()
-    await window.flushAllPromises()
+    await window['flushAllPromises']()
```

Comment:
REVERTED. Was probably problem with some config on my side. This should work because `flushAllPromises` is declared inside [`jestSetup.ts`](https://github.com/influxdata/ui/blob/master/jestSetup.ts)